code/modules/clothing/glasses/hud.dm

```diff
@@ -3,9 +3,8 @@
 	desc = "A heads-up display that provides important info in (almost) real time."
 	flags_1 = null //doesn't protect eyes because it's a monocle, duh
 	var/hud_type = null
-	///Used for topic calls. Just because you have a HUD display doesn't mean you should be able to interact with stuff.
-	var/hud_trait = null
 
+	// NOTE: Just because you have a HUD display doesn't mean you should be able to interact with stuff.
```

Comment:
```suggestion
	// NOTE: Just because you have a HUD display doesn't mean you should be able to interact with stuff on examine, that's where the associated trait (TRAIT_MEDICAL_HUD, TRAIT_SECURITY_HUD, etc) is necessary. 
```